src/compiler/factory/emitHelpers.ts

```diff
@@ -1102,8 +1102,8 @@ const generatorHelper: UnscopedEmitHelper = {
     priority: 6,
     text: `
             var __generator = (this && this.__generator) || function (thisArg, body) {
-                var _ = { label: 0, sent: function() { if (t[0] & 1) throw t[1]; return t[1]; }, trys: [], ops: [] }, f, y, t, g;
-                return g = { next: verb(0), "throw": verb(1), "return": verb(2) }, typeof Symbol === "function" && (g[Symbol.iterator] = function() { return this; }), g;
+                var _ = { label: 0, sent: function() { if (t[0] & 1) throw t[1]; return t[1]; }, trys: [], ops: [] }, f, y, t, g = Object.create((typeof Iterator === "function" ? Iterator : Object).prototype);

```

Comment:
I assume it's not a problem if someone redefines the global `Iterator`, given we don't care if someone does that for other global symbols?

Reply:
Correct